docs/development/setup-vagrant.md

```diff
@@ -1016,6 +1016,22 @@ If at any time you wish to revert back to the default settings, simply
 remove the `GUEST_CPUS` and `GUEST_MEMORY_MB` lines from
 `~/.zulip-vagrant-config`.
 
+
+#### MacOS [os version - sierra and Mojave] - Error while installing VirtualBox
+
+If you receive the following error while installing VirtualBox in MacOS:
+
+```
+The installation failed. The Installer encountered an error that caused the installation to fail.
```

Comment:
Is this the complete error log? This looks like the start of a longer log and could lead to false positives. Someone might be having some other error, but they'd still first assume that it must be because the os is blocking the installation.

Reply:
Actually this error arises during GUI installation of VirtualBox from dmg file.
I am attaching the screenshot here.
<img width="617" alt="Screenshot 2020-01-02 at 5 08 32 PM" src="https://user-images.githubusercontent.com/11742778/71701303-b1ff8200-2dee-11ea-927e-30c8f6b17486.png">
I'll attach this screenshot in the .md file for reference.